config.go

```diff
@@ -76,8 +79,29 @@ type Config struct {
 	SASLPassword  string
 }
 
+func (c *Config) Validate(ctx context.Context) error {
```

Comment:
I wouldn't use the name `Validate` here, just because it doesn't match the SDK documentation for [`Configure`](https://github.com/ConduitIO/conduit-connector-sdk/blob/8477ab1b656aff3bc91c8cac77277f2d736d7bfb/source.go#L36-L40):

> Configure is the first function to be called in a connector. It provides the connector with the configuration that needs to be validated and stored. In case the configuration is not valid it should return an error. Testing if your connector can reach the configured data source should be done in Open, not in Configure.

The doc explicitly states the config needs to be validated, but testing if the data source/destination can be reached should not be part of the validation. And here we are "validating the config" which does exactly that 😅 
 Even though we don't call `Config.Validate` in `Configure` it still is a bit misleading IMO.

Reply:
Good point, I'll rename to `Test`.